redfish-core/lib/network_protocol.hpp

```diff
@@ -359,9 +359,9 @@ inline void
         }
     };
 
-    std::vector<std::string> interfaces = {
```

Comment:
You can see https://gerrit.openbmc.org/c/openbmc/bmcweb/+/59537/12/redfish-core/lib/network_protocol.hpp  to find out which functions are changed.
The changes are seen 'green' in right panel.


The changed functions in gerrit are:
handleNTPServersPatch()
handleProtocolEnabled();

```
248 inline void
249     handleNTPServersPatch(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
250                           const std::vector<nlohmann::json>& ntpServerObjects,
251                           std::vector<std::string> currentNtpServers)
...
323     // Any remaining array elements should be removed
324     currentNtpServers.erase(currentNtpServer, currentNtpServers.end());
325
326+     constexpr std::array<std::string_view, 1> ethInterfaces = {
327+         "xyz.openbmc_project.Network.EthernetInterface"};
328+     dbus::utility::getSubTree(
329+         "/xyz/openbmc_project", 0, ethInterfaces,
330         [asyncResp, currentNtpServers](
331             const boost::system::error_code& ec,
332             const dbus::utility::MapperGetSubTreeResponse& subtree) {
```

```
369 inline void
370     handleProtocolEnabled(const bool protocolEnabled,
371                           const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
372                           const std::string& netBasePath)
373 {
374+     constexpr std::array<std::string_view, 1> interfaces = {
375+         "xyz.openbmc_project.Control.Service.Attributes"};
376+     dbus::utility::getSubTree(
377+         "/xyz/openbmc_project/control/service", 0, interfaces,
378         [protocolEnabled, asyncResp,
379          netBasePath](const boost::system::error_code& ec,
380                       const dbus::utility::MapperGetSubTreeResponse& subtree) {
381         if (ec)
382         {
383             messages::internalError(asyncResp->res);
384             return;
385         }
```